kahuna/public/js/components/gr-panel/gr-panel.css

```diff
@@ -4,7 +4,7 @@
 }
 
 .panel--side {
-    width: 280px;
+    width: 284px;
```

Comment:
Can we document (as a comment) these magic values to people who come across them later know what to watch out for? It also alleviates fear of change if they no longer apply because the reason has gone away.